.chloggen/chan-tim_deprecateCompressEncoding

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
```

Comment:
It is `breaking` or `deprecation` change. We should provide in this PR also migration steps for customers

Reply:
whoops yeah its deprecation 😆 

what do you mean by migration steps? are the customers not using compression already?

Reply:
Yeah, this will be a breaking change.

Reply:
added migration steps in a separate PR: https://github.com/SumoLogic/sumologic-otel-collector/pull/1605